src/shogun/features/SubsetStack.cpp

```diff
@@ -48,6 +48,7 @@ CSubsetStack::CSubsetStack(const CSubsetStack& other)
 		m_active_subset=(CSubset*)other.m_active_subsets_stack->get_element(i);
```

Comment:
I think also maybe this could be an `auto` or a `some` rather than using the member